src/naive/date.rs

```diff
@@ -252,6 +251,18 @@ impl NaiveDate {
         }
     }
 
+    /// Skips the range check on years.
+    ///
+    /// FOR INTERNAL USE ONLY.
+    fn unchecked_from_of(year: i32, of: Of) -> Option<NaiveDate> {
+        if of.valid() {
```

Comment:
Style nit: prefer to invert the condition, to return `None` early.

Reply:
I'll change it to return early. After https://github.com/chronotope/chrono/pull/1043 this check should be gone anyway.